Numsense/ObjectOriented.fs

```diff
@@ -43,10 +43,17 @@ type RussianNumeralConverter () =
         member this.ToNumeral number = Numeral.toRussian number
         member this.TryParse (s, result) =
             Helper.tryParse Numeral.tryParseRussian (s, &result)
+
+type LithuanianNumeralConverter () =
+    interface INumeralConverter with
+        member this.ToNumeral number = Numeral.toLithuanian number
+        member this.TryParse (s, result) =
+            Helper.tryParse Numeral.tryParseLithuanian (s, &result)
 
 type Numeral private () =
     static member val English = EnglishNumeralConverter () :> INumeralConverter
     static member val Danish  =  DanishNumeralConverter () :> INumeralConverter
     static member val Polish  =  PolishNumeralConverter () :> INumeralConverter
     static member val Dutch   =   DutchNumeralConverter () :> INumeralConverter
-    static member val Russian = RussianNumeralConverter () :> INumeralConverter
\ No newline at end of file
+    static member val Russian = RussianNumeralConverter () :> INumeralConverter
+    static member val Lithuanian = LithuanianNumeralConverter () :> INumeralConverter
```

Comment:
Using the Devil's Advocate review technique, if I remove this member, then all tests still pass.